src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -248,13 +262,16 @@ export async function startTransformByQ() {
         transformByQState.setToSucceeded()
         if (status === 'PARTIALLY_COMPLETED') {
             transformByQState.setToPartiallySucceeded()
+            resultStatusMessage = 'JobPartiallySucceeded'
         }
 
         await vscode.commands.executeCommand('aws.amazonq.transformationHub.reviewChanges.reveal')
         await vscode.commands.executeCommand('aws.amazonq.refresh')
         sessionPlanProgress['returnCode'] = StepProgress.Succeeded
+        resultStatusMessage = 'JobCompletedSuccessfully'
     } catch (error) {
         if (transformByQState.isCancelled()) {
+            resultStatusMessage = 'User initiated stop'
```

Comment:
I would change to `UserCancelled` for consistency

Reply:
IntelliJ has "User initiated stop": https://github.com/aws/aws-toolkit-jetbrains/blob/9fd13eb1a8dd9b291a430327e3aba6f2614d610e/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt#L606

Reply:
Oh sorry I mostly just meant using camel case seems to be the standard at least for VS Code: `UserInitiatedStop`

Reply:
I think we can keep camel case for `reason` field, but `codeTransformResultStatusMessage` should be fine to contain regular sentence, which will help with metric aggregation across IDEs.

Reply:
Actually, I'll change it to UserCancelled (for IntelliJ as well)